quiz.js

```diff
@@ -37,8 +50,20 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update(
+  {house: "House Stark"},
+  {$push: {people: ["Ned Stark", "Arya Stark","Sansa Stark"]}}
+  )
 
+db.houses.update(
+  {house: "House Targaryen"},
+  {$push: {people: ["Viserys Targaryen", "Daenerys Targaryen"]}}
+  )
 
+db.houses.update(
+  {house: "House Arryn"},
+  {$push: {people: ["Jon Arryn"]}}
+  )
 
```

Comment:
Close! $push is expecting a single value. By giving it an array, what Mongo will actually do is take your array and make it the first element in the `people` array, creating a nested array.
```
{
	"_id" : ObjectId("..........................................."),
	"house" : "House Stark",
	"motto" : "Winter is Coming",
	"people" : [
		[
			"Ned Stark",
			"Arya Stark",
			"Sansa Stark"
		]
	]
}
```
We could simply enter each person one at a time, but it could get tedious if we had a lot of people to add. Fortunately, there is another way : [$each](http://docs.mongodb.org/manual/reference/operator/update/each/).
```
db.houses.update(
  {house: "House Stark"},
  {$push: {people: {$each: ["Ned Stark", "Arya Stark","Sansa Stark"] } } }
 )
```